Dockerfile

```diff
@@ -4,6 +4,8 @@ USER root
 WORKDIR /app
 ADD . /app
 RUN pip install -r requirements.txt
-RUN cd telemetry_src && python setup.py install; cd ..
+RUN git clone -b artifacts-support https://github.com/sdgtt/telemetry.git
```

Comment:
can we get this branch merged into main before things diverge too much

Reply:
clone  "telemetry:master" instead as telemetry:artifacts-support" was already merged to master https://github.com/sdgtt/telemetry/pull/28

https://github.com/sdgtt/constellation/commit/f02743275ed0c1c14adc1444ce70b52198a8ab12